specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Get_WithDiskControllerType.json

```diff
@@ -0,0 +1,243 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmScaleSetName": "{vmss-name}",
+    "api-version": "2022-08-01",
+    "parameters": {
```

Comment:
Get request doesn't contain the request payload used for create/update, pls. update the request payload.

Reply:
acknowledging